seirmo/figures.py

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    Creates incidence number plots for the app
+
+    """
+    def __init__(self):
+        super(IncidenceNumberPlot, self).__init__()
+
+        # Instantiate an empty plotly figure
+        self._fig = go.Figure()
+
+        """
+        Add a bar plot trace for the given incidence number data
+        in the figure
+
```

Comment:
I like the one sentence description. I would recommend to include a short paragraph below that describes the expected data in more detail and the default key labels, etc. Maybe similar to this
https://erlotinib.readthedocs.io/en/latest/plots.html#erlotinib.plots.PDTimeSeriesPlot ?